content/influxdb/v2.4/get-started/setup.md

```diff
@@ -0,0 +1,332 @@
+    In the InfluxDB initialization process, you created a bucket.
+    You can use that bucket or create a new one specifically for this getting
+    started tutorial. All examples in this tutorial assume a bucket named
+    **get-started**.
+
+    Use the **InfluxDB UI**, **`influx` CLI**, or **InfluxDB API** to create a
+    new bucket.
+
+    {{< tabs-wrapper >}}
+{{% tabs %}}
+[InfluxDB UI](#)
+[influx CLI](#)
+[InfluxDB API](#)
+{{% /tabs %}}
+
+{{% tab-content %}}
+<!------------------------------ BEGIN UI CONTENT ----------------------------->
+
+1.  Visit
+    {{% oss-only %}}[localhost:8086](http://localhost:8086){{% /oss-only %}}
+    {{% cloud-only %}}[cloud2.influxdata.com](https://cloud2.influxdata.com){{% /cloud-only %}}
+    in a browser to log in and access the InfluxDB UI.
+
+2.  Navigate to **Load Data** > **Buckets** using the left navigation bar.
+3.  Click **+ {{< caps >}}Create bucket{{< /caps >}}**.
+4.  Provide a bucket name (get-started) and select {{% caps %}}Never{{% /caps %}}
+    to create a bucket with an infinite [retention period](/influxdb/v2.4/reference/glossary/#retention-period).
+5.  Click **{{< caps >}}Create{{< /caps >}}**.
+
+<!------------------------------- END UI CONTENT ------------------------------>
+{{% /tab-content %}}
+{{% tab-content %}}
+<!---------------------------- BEGIN CLI CONTENT ----------------------------->
+
+1.  If you haven't already, [download, install, and configure the `influx` CLI](/influxdb/v2.4/tools/influx-cli/).
+2.  Use the [`influx bucket create` command](/influxdb/v2.4/reference/cli/influx/bucket/create/)
+    to create a new bucket.
+    
+    **Provide the following**:
+
+    - `-n, --name` flag with the bucket name.
+
+    {{< cli/influx-creds-note >}}
+
+    ```sh
+    influx bucket create --name get-started
+    ```
+
+<!------------------------------ END CLI CONTENT ------------------------------>
+{{% /tab-content %}}
+{{% tab-content %}}
+<!----------------------------- BEGIN API CONTENT ----------------------------->
+
+To create a new bucket using the InfluxDB HTTP API, send a request to
+the InfluxDB API `/api/v2/buckets` endpoint using the `POST` request method.
+
+{{< api-endpoint endpoint="http://localhost:8086/api/v2/buckets" method="post" >}}
+
+Include the following with your request:
+
+- **Headers**:
+  - **Authorization**: Token <INFLUX_TOKEN>
+  - **Content-Type**: application/json
+- **Request body**: JSON object with the following properties:
+  - **org**: InfluxDB organization name
+  - **name**: Bucket name
+  - **retentionRules**: List of retention rule objects that define the bucket's retention period.
+    Each retention rule object has the following properties:
+    - **type**: "expire"
+    - **everySeconds**: Retention period duration in seconds.
+      `0` indicates the retention period is infinite. 
+
+```sh
+export INFLUX_HOST=http://localhost:8086
+export INFLUX_ORG_ID=<YOUR_INFLUXDB_ORG_ID>
+export INFLUX_TOKEN=<YOUR_INFLUXDB_API_TOKEN>
```

Comment:
These are already set for OSS by the setup process, but if we're telling them to set INFLUX_TOKEN again, should we specify the all access token they just created or leave it to them to decide?